colcon_bundle/verb/_archive_generators.py

```diff
@@ -73,3 +61,88 @@ def generate_archive(install_base,
     os.remove(bundle_tar_path)
 
     logger.info('Archiving complete')
+
+
+def generate_archive_v2(install_base,
+                        staging_path,
```

Comment:
staging_path -> dependencies_staging_path for consistency with workspace_staging_path?

Reply:
Yep, since we are adding the workspace staging page I think that help disambiguate.